showcase/openfe_showcase.ipynb

```diff
@@ -2,12 +2,11 @@
  "cells": [
```

Comment:
<p>Whilst we're here - OpenFF vs openff, we should stick to one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>